openpype/hosts/maya/tools/mayalookassigner/commands.py

```diff
@@ -122,7 +124,12 @@ def create_asset_id_hash(nodes):
                 pid = k.split(":")[0]
                 if node not in node_id_hash[pid]:
                     node_id_hash[pid].append(node)
-
+        elif shapes and cmds.nodeType(shapes[0]) == "aiStandIn":
+            path = arnold_standin.get_standin_path(shapes[0])
+            for id, _ in arnold_standin.get_cbid_by_node(path).items():
+                pid = id.split(":")[0]
+                if shapes[0] not in node_id_hash[pid]:
```

Comment:
This not in node_id_hash[pid]` lookup will get very slow if there are many nodes per `pid`. Should we refactor this `node_id_hash` to be `defaultdict(set)`?

Also, I have no idea what `pid` stands for. It should be the asset's id. Why not pick `asset_id` as variable name? I know the v-ray logic above also names it `pid` but it's confusing what it refers to right?

Reply:
Yup, have replaced dict with defaultdict where I could find it needed.